__tests__/integration.test.js

```diff
@@ -88,4 +88,55 @@ describe('GET /api/articles/:article_id', () => {
             expect(msg).toBe('invalid ID')
         })
     })
+})
+
+
+describe('GET /api/articles', () => {
+    test('GET200: endpoint responds with an array of the correct numbner of article objects with correct properties', () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body })=>{
+            const { articles } = body
+            
+            expect(articles.length).toBe(13)
+
+            articles.forEach((article)=>{
+                expect(article).toMatchObject({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
+                    topic: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String),
+                    comment_count: expect.any(Number)
+                })
+
+                expect(article.body).toBe(undefined)
+            })
+        })
+    })
+
+    test('GET200: endpoint responds with the articles sorted in descending order', () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body })=>{
+            const { articles } = body
+            
+            expect(articles).toBeSortedBy('created_at', {descending: true})
+        })
+    })
+
+    
+    test('GET404: endpoint responds with appropriate error if endpoint misspelt', () => {
```

Comment:
No need for this last test to be here, its not related to the functionality of this endpoint. All bad paths need only be tested for once and will be dealt with separately to endpoint logic. Might have auto passed? Which is our clue 👍